README_TEMPLATE.md

```
@@ -0,0 +1,334 @@
+        ┌─────────────┐  ┌──────────────┐
+        │ Specialist  │  │ Tips/Journal │
+        │ Directory   │  │ (Stable Path)│
+        └─────────────┘  └──────────────┘
+```
+
+---
+
+## API Documentation
+
+### Base URL
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)